src/main/java/com/ss/training/utopia/agent/controller/AgentController.java

```diff
@@ -36,4 +38,67 @@
 		return new ResponseEntity<Booking[]>(bookingArray, status);
 	}
 
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+	@PutMapping(path="/booking")
```

Comment:
We need to unify our api paths. for example all my paths begin with /utopia/traveler at the moment